pkg/notes/notes.go

```diff
@@ -68,6 +72,13 @@ type CVEData struct {
 	Description string  `json:"description"`
 }
 
+// MajorTheme struct to define the release major versions
+type MajorTheme struct {
```

Comment:
I thought about making it more generic, but I really like the way we can describe it in markdown. :+1: